integration_tests/features/WalletRecovery.feature

```diff
@@ -1,15 +1,49 @@
 @wallet-recovery
 Feature: Wallet Recovery
 
-    @critical
-    Scenario: Wallet recovery with connected base node staying online
-        Given I have a seed node NODE
-        And I have 1 base nodes connected to all seed nodes
-        And I have wallet WALLET_A connected to all seed nodes
-        And I have a merge mining proxy PROXY connected to NODE and WALLET_A with default config
-        When I merge mine 10 blocks via PROXY
-        When I mine 10 blocks on NODE
-        When I wait for wallet WALLET_A to have at least 55000000000 uT
-        Then all nodes are at height 20
-        When I recover wallet WALLET_A into wallet WALLET_B connected to all seed nodes
-        Then wallet WALLET_A and wallet WALLET_B have the same balance
+@critical
+Scenario: Wallet recovery with connected base node staying online
+    Given I have a seed node NODE
+    And I have 1 base nodes connected to all seed nodes
+    And I have wallet WALLET_A connected to all seed nodes
+    And I have a merge mining proxy PROXY connected to NODE and WALLET_A with default config
+    When I merge mine 10 blocks via PROXY
+    When I mine 5 blocks on NODE
+    When I wait for wallet WALLET_A to have at least 55000000000 uT
+    Then all nodes are at height 15
+    When I recover wallet WALLET_A into wallet WALLET_B connected to all seed nodes
+    Then wallet WALLET_A and wallet WALLET_B have the same balance
+    And I have wallet WALLET_C connected to all seed nodes
+    And I send 100000 uT from wallet WALLET_B to wallet WALLET_C at fee 100
+    When I mine 5 blocks on NODE
+    Then all nodes are at height 20
+    Then I wait for wallet WALLET_C to have at least 100000 uT
+
+@critical
+Scenario: Scan for one-sided payments
```

Comment:
Maybe the name should change here as the test is about recovery not scanning. With the latter, I would expect WALLET_B to be started up again in its normal mode and being issued a command to scan for one-sided transactions.

Reply:
Sure, can change the name. In the end recovery is the same process of scanning and this is in the Recovery Feature but 🤷‍♂️

Reply:
Just a **NIT** though

Reply:
Let me mark it as such